src/Npgsql/Npgsql.csproj

```diff
@@ -11,6 +11,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" />
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" />
```

Comment:
A reference already exists in `DirectoryBuild.props`.

Reply:
This was because we have the versions in Directory.Build.targets (note that the PackageReference there has Update), but the actual reference in the csproj (with Include)